app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,48 @@
+class CommentsController < ApplicationController
+  before_action :set_comment, only: [:edit, :update, :destroy]
+
+  def create
+    authorize! :create, Comment
+
+    @comment = Comment.new(comment_params)
+
+    if @comment.save
```

Comment:
We should have a helper method for this.
```ruby
def method(record)
  if record.save
    flash.notice = t("success.#{record.class_name}")
  else
   flash.alert = record.errors.full_messages.to_sentence
  end
end
```

Sth like this. I see a lot of duplication can be saved by this.